lib/interface-types/src/lib.rs

```diff
@@ -0,0 +1,162 @@
+pub mod ast;
+#[macro_use]
+mod macros;
+pub mod decoders;
+pub mod encoders;
+pub mod instructions;
+
+pub use decoders::binary::parse as parse_binary;
+
+#[cfg(test)]
+mod tests {
+    use crate::{ast::*, instructions::Instruction, parse_binary};
+    use std::fs;
+    use wasmer_clif_backend::CraneliftCompiler;
+    use wasmer_runtime_core as runtime;
+
+    fn get_module() -> runtime::Module {
+        runtime::compile_with(
+            fs::read("tests/assets/hello_world.wasm")
+                .expect("Failed to read `tests/assets/hello_world.wasm`.")
+                .as_slice(),
+            &CraneliftCompiler::new(),
```

Comment:
Can we use `compile` instead of `compile_with` ?

```rust
        runtime::compile(
            fs::read("tests/assets/hello_world.wasm")
                .expect("Failed to read `tests/assets/hello_world.wasm`.")
                .as_slice()
        )
```

Reply:
I'm not sure those tests will remain here. I'll keep your comment open for the moment.